src/Analysis/GWASV2/GWASContainer.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from "react";
+import { PropTypes } from "prop-types";
+import { Space, Button, Steps, Step, Popconfirm } from "antd";
+
+const GWASContainer = () => {
+    const [current, setCurrent] = useState(0);
+    const gwasSteps = [
+        {
+            title: 'Step 1',
+            description: 'Select Study Population',
+        },
+        {
+            title: 'Step 2',
+            description: 'Select Outcome Phenotype',
+        },
+        {
+            title: 'Step 3',
+            description: 'Select Covariate Phenotype',
```

Comment:
I think "Select Covariate Phenotypes" will be better, as the user can select multiple covariate phenotypes here

Reply:
true